README.md

```diff
@@ -111,3 +111,17 @@ Once all the dependencies are installed:
 make
 sudo make install
 ```
+
+### Install with Ansible
+
+An [Ansible Role](https://github.com/alvistack/ansible-role-crun) is also available to automate the above steps:
```

Comment:
would you like to add these scripts to the crun repository and maintain them here?  We could add them under `contrib/ansible`.  What do you think?

Reply:
Ummm... We think about that once before for CRI-O (see https://github.com/cri-o/cri-o/pull/3628#discussion_r413565850)...

My original target for this PR is to provide a hints for people with DevOps knowledge to kickstart with their own integration works.

To be honest the role will not works in solo: it also depends on other roles, e.g. golang, git, runc, conmon, etc (see https://github.com/alvistack/ansible-role-crun/blob/master/ansible-galaxy-requirements.yml).

Why the steps look such simple now, just because molecule will fetch the dependency before the converge start. So the truth is you will have more than single Ansible Role being used. IMHO, in this repo we may not hope to locally embed too many Ansible Role dependency, it shouldn’t be this repo target.

Moreover for production deployment use cases, you may hope to use my Ansible Collection for Kubernetes (see https://github.com/alvistack/ansible-collection-kubernetes/blob/master/playbooks/cri_o-install.yml), for multi target DevOps management. BTW, promote the Ansible Collection here may again overkill.

Reply:
ok, then I am fine with the current version.

Mine was just a suggestion so we could centralize all the crun stuff in the same repository.